genutil/doc.go

```diff
@@ -14,5 +14,5 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package utils implements utils code.
-package utils
+// Package genutil implements general utility code.
+package genutil
```

Comment:
let us avoid having anything in the root folder, see if we can move this under pkg to internal